android-live-javacv/app/src/main/java/com/example/mmcc/javacvexample/MainActivity.java

```diff
@@ -0,0 +1,531 @@
+package com.example.mmcc.javacvexample;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.ActivityInfo;
+import android.hardware.Camera;
+import android.hardware.Camera.PreviewCallback;
+import android.media.AudioFormat;
+import android.media.AudioRecord;
+import android.media.MediaRecorder;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Display;
+import android.view.KeyEvent;
+import android.view.LayoutInflater;
+import android.view.SurfaceHolder;
+import android.view.SurfaceView;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.ShortBuffer;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import org.bytedeco.javacpp.avutil;
+import org.bytedeco.javacv.FFmpegFrameFilter;
+import org.bytedeco.javacv.FFmpegFrameRecorder;
+import org.bytedeco.javacv.Frame;
+import org.bytedeco.javacv.FrameFilter;
+
+public class MainActivity extends Activity implements OnClickListener {
+
+    private final static String CLASS_LABEL = "MainActivity";
+    private final static String LOG_TAG = CLASS_LABEL;
+
+    /* This isn't a live RTMP endpoint. You should replace this line with your own! */
+    private String ffmpeg_link = "rtmp://live.mux.com/app/f5212894-e8ee-9eba-02e9-ac21c6249e0b";
```

Comment:
you deleted this live stream, right?

Reply:
May want to put in like a `YOUR_STREAM_KEY`

Reply:
Yeah that stream is deleted, but I updated the name.